### ISSUE TITLE
Update to org.freedesktop.Platform//22.08 blocked by failing build of gvfs module

### ISSUE THREAD

**Issue by stbergmann** [flathub/org.libreoffice.LibreOffice#198]
At <https://github.com/flathub/org.libreoffice.LibreOffice/pull/195/commits/e9da500c6d377d251368ee77b245a760e4ec0543> "Update to Freedesktop SDK 22.08" I try to update from 21.08, but at least my local test builds would then start to fail with
```
========================================================================
Building module gvfs in /data/sbergman/lo-flatpak-builder/.flatpak-builder/build/gvfs-2
========================================================================
FB: Running: tar xf /data/sbergman/lo-flatpak-builder/.flatpak-builder/downloads/ed136a842c996d25c835da405c4775c77106b46470e75bdc242bdd59ec0d61a0/gvfs-1.38.1.tar.xz --no-same-owner --strip-components=1 -J
FB: Running: flatpak build --die-with-parent --env=FLATPAK_BUILDER_BUILDDIR=/run/build/gvfs --nofilesystem=host:reset --filesystem=/data/sbergman/lo-flatpak-builder/.flatpak-builder/build/gvfs-2 --bind-mount=/run/build/gvfs=/data/sbergman/lo-flatpak-builder/.flatpak-builder/build/gvfs-2 --build-dir=/run/build/gvfs/_flatpak_build --bind-mount=/run/ccache=/data/sbergman/lo-flatpak-builder/.flatpak-builder/ccache --env=SOURCE_DATE_EPOCH=1665411192 '--env=CFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer ' '--env=CXXFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer ' '--env=LDFLAGS=-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed ' --env=CCACHE_DIR=/run/ccache --env=PATH=/run/ccache/bin:/app/bin:/usr/bin --env=LD_LIBRARY_PATH=/app/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig --env=FLATPAK_BUILDER_N_JOBS=12 /data/sbergman/lo-flatpak-builder/.flatpak-builder/rofiles/rofiles-BWr7Jp meson --prefix=/app .. -Dsystemduserunitdir=no -Dtmpfilesdir=no -Dinstalled_tests=true -Ddbus_service_dir=/usr/share/dbus-1/services -Dadmin=false -Dafc=false -Dafp=false -Darchive=false -Dcdda=false -Ddnssd=false -Dgoa=false -Dgoogle=false -Dgphoto2=false -Dhttp=false -Dmtp=false -Dnfs=false -Dsftp=false -Dsmb=false -Dudisks2=false -Dbluray=false -Dfuse=false -Dgcr=false -Dgcrypt=false -Dgudev=false -Dkeyring=false -Dlogind=false -Dlibusb=false
The Meson build system
Version: 0.63.2
Source dir: /run/build/gvfs
Build dir: /run/build/gvfs/_flatpak_build
Build type: native build

../meson.build:1:0: ERROR: Unknown options: "dbus_service_dir"

A full log can be found at /run/build/gvfs/_flatpak_build/meson-logs/meson-log.txt
Error: module gvfs: Child process exited with code 1
```
That Meson-based build of gvfs had been introduced with <https://github.com/flathub/org.libreoffice.LibreOffice/commit/7c0a9d3fcd5988bc955bf5d67aa3249d45727bb4> "Update gvfs to 1.38.1".  It's unclear to me why that starts to fail now and how to fix it.  Maybe somebody else (@Erick555 ?) has an idea...

**Comment by Erick555** [comment]
New meson version which is available in 22.08 started to error out on unknown options passed to it and indeed [dbus_service_dir](https://gitlab.gnome.org/GNOME/gvfs/-/blob/master/meson_options.txt) doesn't seem to exist.

Removing `dbus_service_dir` should fix this.

**Comment by stbergmann** [comment]
thanks, that fixed it
